openedx/core/djangolib/tests/test_js_utils.py

```diff
@@ -65,6 +68,31 @@ def test_dump_js_escaped_json_with_custom_encoder_escapes_unsafe_html(self):
         escaped_json = dump_js_escaped_json(malicious_dict, cls=self.SampleJSONEncoder)
         self.assertEquals(expected_custom_escaped_json, escaped_json)
 
+    def test_dump_js_escaped_json_with_lazy_text(self):
```

Comment:
Unit tests are great. Please also make the platform break by activating the same feature via integration testing, much like it's done here: https://github.com/edx/edx-platform/pull/16019/files